.eslintrc.json

```diff
@@ -20,6 +20,7 @@
   "plugins": ["react"],
   "rules": {
     "no-param-reassign": 0,
+    "no-console": [1, { "allow": ["error"] }],
```

Comment:
@dusmel Why did you add this rule?

Reply:
Because right now we don't have a way of handling errors and it was giving me a hard time to debug so I added a rule to allow only console.error to know why codes aren't behaving the way they are supposed to!